back-end/api/requirements.txt

```diff
@@ -0,0 +1,29 @@
+asgiref==3.3.1
+certifi==2020.12.5
+cffi==1.14.4
```

Comment:
I had problems with the requirements.txt too when I dabbled getting it to work with Heroku, so as long as the app deploys automatically I'm personally good with it. We just need to be careful as a team if we install new packages

Reply:
Sounds good!